sample/EasyTables/CSharp/QueueToCollector/function.json

```diff
@@ -0,0 +1,15 @@
+﻿{
+  "disabled": true,
+  "bindings": [
+    {
+      "type": "queueTrigger",
+      "direction": "in",
+      "queueName": "samples-workitems"
```

Comment:
I was only running one at a time (hence all of them being disabled). And only loading EasyTable scripts. I saw elsewhere that you have a new way to turn on one function at a time -- I'll look at that.

---
In reply to: [56186001](https://github.com/Azure/azure-webjobs-sdk-script/pull/98#discussion_r56186001) [](ancestors = 56186001)